Doppler.HtmlEditorApi/Storage.DapperProvider/LoadCampaignQuery.cs

```diff
@@ -0,0 +1,46 @@
+using System.Data;
+using System.Threading.Tasks;
+using Dapper;
+
+namespace Doppler.HtmlEditorApi.Storage.DapperProvider;
+
+public static class LoadCampaignQuery
+{
+    private const string QUERY = @"
+SELECT
+    CAST (CASE WHEN co.IdCampaign IS NULL THEN 0 ELSE 1 END AS BIT) AS CampaignHasContent,
+    CAST (CASE WHEN ca.IdUser IS NULL THEN 0 ELSE 1 END AS BIT) AS CampaignBelongsUser,
+    CAST (CASE WHEN ca.IdCampaign IS NULL THEN 0 ELSE 1 END AS BIT) AS CampaignExists,
+    ca.IdCampaign,
+    co.Content,
+    co.EditorType,
+    co.Meta
+FROM [User] u
+LEFT JOIN [Campaign] ca ON
+    u.IdUser = ca.IdUser
+    AND ca.IdCampaign = @IdCampaign
+LEFT JOIN [Content] co ON
+    ca.IdCampaign = co.IdCampaign
+WHERE
+    u.Email = @AccountName";
+
+    public static Task<Result> QueryFirstOrDefaultAsync(this IDbContext dbContext, Parameters param)
```

Comment:
Ahora que tengo contexto le pondría 
```suggestion
    public static Task<Result> FirstOrDefaultAsync(this IDbContext dbContext, Parameters param)
```
Luego en la implementación le estamos pasando la Query que podría tener un nombre mas especifico con lo que va a retornar.

Reply:
Bueno, teniendo en cuenta esto y otras ideas de @ms-darianbenito cambié el approach, creo que ahora los nombres son mejores y sin el parche de la clase estática también estamos mejor.

El patron de query también es conocido, así que creo que es más intuitivo.

Resulevo la discución.